public/resources/js/app.js

```diff
@@ -47,6 +47,153 @@ const app = {
     }
   },
   
+  reviewPosts: function( id ) {
+    let params = new URLSearchParams();
+    params.append('id', id);
+    Swal.fire({
+      didOpen: () => {
+        Swal.showLoading();
+        fetch('/posts/show_json/', {
+          method: 'POST',
+          body: params,
+        }).then(response => response.json())
+        .then(data => {
+          console.log(data)
+          Swal.hideLoading();
+          Swal.update({
+            html: `
+            <h1>REVIEW POST</h1>
+            <div class="adminMenu-header">
+              <div class="adminMenu-showpost">
+                <div class="adminMenu-show">
+                  <div class="user_card">
+                      <img src="/resources/img/user.png" alt="user" class="user-card-img">
+                      <div>
+                        <p class="profile-card">${data.username}</p>
+                        <p class="date">${data.created_at}</p>
+                      </div>
+                    <p class="user_card-post_theme">
+                        <i class="${data.theme_icon}"></i>
+                        ${data.theme}">
+                    </p>
+                  </div>
+                  <div class="line"></div>
+                  <div>
+                    <h2>${data.title}</h2>
+                    <p>${data.description}</p>
+                    <div class="images">
+                    `+ (data.images && data.images.length > 0 ?`
+                      <div class="image">
+                        <img src="/assets/imgs/${data.images[0].image}" alt='Image from "${data.title}"' onclick="openModal(${data.id})">
+                          `+ (data.images.length > 1 ? `
+                            <div class="image-overlay" onclick="openModal(${data.id})">+${data.images.length - 1}</div>
+                          `: '') + `
+                      </div>
+                    `: '') + `
```

Comment:
avoid if possible use ternary operator and string concatenation at the same time in this context, even if this could be better for save some lines, in terms of readability this is the worst thing possible, when a project is growing up usually you return to the features you or others made, this could lead in an unnecessary overexertion to analyze the code